l1-contracts/script/UpgradeMainnetUniFiAVS.s.sol

```diff
@@ -22,11 +23,12 @@ contract UpgradeMainnetUniFiAVS is BaseScript {
         address opsWallet = 0xC0896ab1A8cae8c2C1d27d011eb955Cca955580d;
```

Comment:
If you want to, you can refactor this script to use https://github.com/PufferFinance/puffer-contracts/blob/master/mainnet-contracts/script/DeployerHelper.s.sol

Just copy & paste the file, and you will not need to hardcode addresses like this.

Here is an example of how I used it. https://github.com/PufferFinance/puffer-contracts/blob/master/mainnet-contracts/script/UpgradeValidatorTicket.s.sol